### PR TITLE
[ACS-7646] Bump guava version to 33.1.0-jre

### PR DESCRIPTION
**Pull request by aonych**
Bumping:
- guava version to **33.1.0-jre**
- webscripts version to **8.50**

Full dependency tree: [dependency_tree.log](https://github.com/Alfresco/alfresco-community-repo/files/15130363/dependency_tree.log)

### CODE REVIEW

**Review by dsibilio — APPROVED** [review]

File: repository/pom.xml [hunk 1]
```diff
@@ -83,6 +83,7 @@
         <dependency>
             <groupId>commons-fileupload</groupId>
             <artifactId>commons-fileupload</artifactId>
+            <version>1.5</version>
```
Comment (dsibilio) [inline repository/pom.xml:86]: Unnecessary as this is handled by the `dependencyManagement` section of the pom.

File: pom.xml [hunk 2]
```diff
@@ -397,7 +398,7 @@
             <dependency>
                 <groupId>commons-fileupload</groupId>
                 <artifactId>commons-fileupload</artifactId>
-                <version>1.4</version>
+                <version>1.5</version>
```
Comment (dsibilio) [inline pom.xml:401]: Unnecessary change as this is already on `release/7.3.N`, pull the latest `release/7.3.N` instead after reverting this change.

**Review by dsibilio — CHANGES_REQUESTED** [review]

File: repository/pom.xml (hunk 1 above)
Comment (aonych) [reply repository/pom.xml]: Revert 👍

File: pom.xml (hunk 2 above)
Comment (aonych) [reply pom.xml]: Revert 👍

**Review by dsibilio — APPROVED** [review]
